docs/advanced/additional-requests.rst

```diff
@@ -227,13 +227,13 @@ take a look at another example having an HTML response:
     response = web_poet.HttpResponse(
         url="https://www.api.example.com/product-pagination/",
         body=(
-            '<html>'
-            '  <head>'
-            '    <title>Some page</title>'
-            '    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">'
-            '  </head>'
-            '  <body>Sample content 💯</body>'
-            '</html>'
+            "<html>"
+            "  <head>"
+            "    <title>Some page</title>"
+            "    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">"
```

Comment:
this doesn't work, because `"` is inside a string

Reply:
Good catch! Fixed in https://github.com/scrapinghub/web-poet/pull/61/commits/c420e01de6eea71832d49064ad402d30908112ba